index.html

```diff
@@ -75,269 +85,456 @@ <h2 class="intro-heading">I'am a Software Developer</h2>
+                  id="recent-button-1"
+                  >CSS</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-2"
+                  >HTML</a
+                >
+              </li>
+              <li>
+                <a
+                  href="
+ https://getbootstrap.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-3"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-4"
+                  >Ruby</a
+                >
+              </li>
+            </ul>
             <a
-              href="https://www.w3.org/Style/CSS/"
-              class="recent-button active-btn-effect hover-btn-effect"
-              id="recent-button-1"
-              >CSS</a
+              href=""
+              class="btn-see-project active-btn-effect hover-btn-effect"
+              >See project</a
             >
-          </li>
-          <li>
+          </div>
+        </div>
+        <div class="recent-project-wraper">
+          <div class="project">
+            <h3 class="project-title">Profesional Art Printing Data</h3>
+            <p class="project-details">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="project-technology">
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="project-btn project-btn-1 active-btn-effect hover-btn-effect"
+                >
+                  HTML</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  class="project-btn project-btn-2 active-btn-effect hover-btn-effect"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  >Ruby</a
+                >
+              </li>
+            </ul>
             <a
-              href="https://html.com/"
-              class="recent-button active-btn-effect hover-btn-effect"
-              id="recent-button-2"
-              >HTML</a
+              href="#"
+              class="see-project-btn active-btn-effect hover-btn-effect"
+              >See project</a
             >
-          </li>
-          <li>
+          </div>

```

Comment:
- [ ] Cards should also have the same height. As it is the first white card looks slightly longer than the second one, this is because the height property is set to `auto`. Kindly give all the cards the uniform given height of `435px` for a more uniform design.

Your design
![image](https://github.com/FilbertKimasa/Portfolio/assets/81932478/cd9db7e3-5758-498a-a267-4d8b7af80c85)

Given design
![image](https://github.com/FilbertKimasa/Portfolio/assets/81932478/3823bb54-069b-4f7f-a087-51cb2b542e6f)